### ISSUE TITLE
HA-Bridge and V4.1

### ISSUE THREAD

**Issue by pponce** [ebaauw/homebridge-hue#107]
I believe 4.1 release broke my ability have my harmony activities that HA-Bridge exposes as lights.

I get this error when starting with a fresh config and letting homebridge hue find my ha-bridge "hue" hub.
[5/3/2017, 5:12:50 PM] 192.168.1.99: hue bridge request #1: get philipshue/config
[5/3/2017, 5:12:50 PM] 192.168.1.99: hue bridge http status 404

I get the same error when i use my old working user id and specify the host ip directly.
[5/3/2017, 5:18:43 PM] [HA-Bridge] 192.168.1.99: hue bridge request #1: get philipshue/config
[5/3/2017, 5:18:43 PM] [HA-Bridge] 192.168.1.99: hue bridge http status 404

any ideas?

**Comment by ebaauw** [comment]
Oops, looks like I broke v0.4.1.  I accidentally published a hack I needed to get deCONZ working.  Will fix that asap.

I also started whitelisting bridges.  You'll probably get an unknown bridge warning after the (first) fix.  Please post that message here and I'll add the HA-Bridge.

**Comment by ebaauw** [comment]
> I also started whitelisting bridges. You'll probably get an unknown bridge warning after the (first) fix. Please post that message here and I'll add the HA-Bridge.

You already posted the HA-Bridge config in issue #45.  It identifies itself with `modelid` BSB002,  so homebridge-hue will think it's a regular v2 Hue bridge.  No need to whitelist it separately.

**Comment by ebaauw** [comment]
> Oops, looks like I broke v0.4.1.Fixed in v0.4.22.

**Comment by pponce** [comment]
Thanks for looking into this.
With latest version i'm getting this error now.

with no host set or user name set:
[5/4/2017, 7:26:18 AM] 192.168.1.99: hue bridge request #1: get /config
[5/4/2017, 7:26:18 AM] 192.168.1.99: hue bridge http status 500

with proper configuration of host IP and last working user name:
[5/4/2017, 6:47:03 AM] [HA-Bridge] 192.168.1.99: hue bridge request #1: get /config
[5/4/2017, 6:47:03 AM] [HA-Bridge] 192.168.1.99: hue bridge http status 500

So something has changes but still getting an error.
let me know if i can run and any commands to get you more info,

**Comment by pponce** [comment]
Oh and BSB002 is what HA-Bridge appears to report spontaneously. I had nothing to do with that model ID.

**Comment by ebaauw** [comment]
HTTP status 500 is an internal server error.  Unlike the 4xx statuses, the 5xx statuses aren't caused by (shouldn't be reported because of) an invalid client request.  Did you try and restart the HA-Bridge?

**Comment by pponce** [comment]
Ok. i'm back in business. 
HA-Bridge had also recently added some security options. I upgraded homebridge-hue and ha-bridge together. 
https://github.com/bwssytems/ha-bridge/wiki/Security-Configuration
HA-bridge now Included an enable link button option (default off) and an option to use or not use username and pass for hue api (default to not use HUE api username and pass).
I enabled both and all looks well now. The hue api user name and pass option not being on was my issue after you fixed homebridge-hue.

thanks!
